relayer/config/viper.go

```diff
@@ -32,17 +32,11 @@ func BuildViper(fs *pflag.FlagSet) (*viper.Viper, error) {
 	v.AutomaticEnv()
 	// Map flag names to env var names. Flags are capitalized, and hyphens are replaced with underscores.
 	v.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
-	if err := v.BindPFlags(fs); err != nil {
-		return nil, err
-	}
 
-	// Verify that required flags are set
-	if !v.IsSet(ConfigFileKey) {
-		DisplayUsageText()
-		return nil, fmt.Errorf("config file not set")
+	filename, err := fs.GetString(ConfigFileKey)
```

Comment:
this is the only usage of the passed in flagset variable here which means it should probably be pulled out. Didn't do it just yet because it would require unit test changes but I can